src/launchpad/artifacts/apple/zipped_xcarchive.py

```diff
@@ -295,8 +295,8 @@ def get_asset_catalog_details(self, relative_path: Path) -> List[AssetCatalogEle
                 data = json.load(f)
 
             return [self._parse_asset_element(item, parent_path) for item in data]
-        except Exception as e:
-            logger.error(f"Failed to get asset catalog details for {relative_path}: {e}")
+        except Exception:
+            logger.exception(f"Failed to get asset catalog details for {relative_path}")
```

Comment:
I'll remove the paths in here once we get more users, right now seems worth it to create new issues.